owtf/data/conf/profiles/plugin_web/groups.cfg

```diff
@@ -75,3 +75,4 @@ OWTF-WVS-006 | 99 | Skipfish Unauthenticated |  | http://code.google.com/p/skipf
 OWTF-WSP-001 | 99 | Visit URLs | Visit urls found by scumbag spidering | http://owtf.org
 OWTF-CL-001 | 99 | Public Buckets | Open Amazon S3 Buckets | https://grayhatwarfare.com
 OWTF-ST-001 | 99 | Subdomain Takeover | Test for Subdomain Takeover  | https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/10-Test_for_Subdomain_Takeover
+OWTF-CL-002 | 99 | Open Buckets | Open Google Cloud Platform Buckets | https://rhinosecuritylabs.com/gcp/google-cloud-platform-gcp-bucket-enumeration/
```

Comment:
can you rename this to "Public buckets" to match with the AWS one above?

Reply:
done.